services/app-web/src/pages/CMSDashboard/RateReviewsDashboard/RateReviewsTable.tsx

```diff
@@ -256,6 +311,7 @@ export const RateReviewsTable = ({
                 meta: {
                     dataTestID: `${tableConfig.rowIDName}-date`,
                 },
+                filterFn: 'endDateFilter',
```

Comment:
🐛 This is the bug - as discussed in zoom. This end date filter is not a filter for the rate end date column. It should be applied along side the the start date filter for the rating start date column (we are filtering that a rating start date is within/between a range)